### ISSUE TITLE
paw not moving 

### ISSUE THREAD

**Issue by VacCatGT** [kuroni/bongocat-osu#101]
yeah yeah i have fullscreen and windows 7 compatibility mode on and i just alt tab to osu but the paw doesnt move while ingame

**Comment by HorstiKruste** [comment]
I have kinda the same problem here as well.
using following things:
win10
compatible mode for win7 (both tested)
v1.51
not minimazed
on same screen which one i want to "scan" with mouse (mouse works totally fine)
League of Legends (I see people like https://www.twitch.tv/videos/818351435?t=02h37m48s using it without any problems)

Programm working fine in OBS and in browser.
when i tab into LoL-Client or LoL-Game (they are two seperate .exe so I list both of them) the programm stop recording/recognizing keystrokes, while mouse keeps working fine.

I know "not Osu! and so on... but maybe this info help and maybe gave all of us LoL-players a solution for this problem :)

Thanks in advance

KR
Horsti

**Comment by VacCatGT** [comment]
yeah i also tried the compability modes and i have the exact same problem.
when i alt tab it just freezes. the solution (if we find one) is probably
the same for every game

**Comment by HorstiKruste** [comment]
found the solution for this problem.

execute the programm with admin rights "run as admin" or set it up in the properties.

I expect that you execute ur game as an admin as well...

This was the way it works for me.
please give feedback so we can close this issue. and at Kuroni, maybe leave a hint in the readme for this setting.

KR
Horsti

**Comment by VacCatGT** [comment]
ah im pretty sure i tried that before but ill try it again to see if it
works. ill close the issue if it worked. thanks!

**Comment by kuroni** [comment]
I can't provide any useful information really, never ran into that problem :(
